core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -3614,6 +3614,12 @@ object Stream extends StreamLowPriority {
   def resource[F[_], O](r: Resource[F, O])(implicit F: MonadCancel[F, _]): Stream[F, O] =
     resourceWeak(r).scope
 
+  /** Same as [[resource]], but expressed as a FunctionK. */
+  def resourceK[F[_]](implicit F: MonadCancel[F, _]): Resource[F, *] ~> Stream[F, *] =
```

Comment:
**suggestion (non-blocking):** Should there be a `resourceWeakK` too?

Reply:
don't see why not